### PR TITLE
Documentation per issue #213.

### PR DESCRIPTION
**Pull request by BHagedorn-IDM**

### CODE REVIEW

File: pacehrh/vignettes/pacehrh.rmd [hunk 1]
```diff
@@ -141,15 +141,15 @@ stochastic distributions around various parameter values. See the table below fo
...
+| Incidence rates <br />(**TaskValues** column H)               | Uniform                  | mean, p          | $$min = mean - mean*p$$ $$max = mean + mean*p$$ |
+| Annual delta fertility rates                                  | Truncated normal*        | delta, p, q      | $$mean = delta$$ $$std.dev. = p$$ $$a = mean - p*q$$ $$b = mean + p*q$$  |
+| Annual delta mortality rates                                  | Truncated normal*        | delta, p, q      | $$mean = delta$$ $$std.dev. = p$$ $$a = mean - p*q$$ $$b = mean + p*q$$ | 
+| Annual delta incidence rates <br />(**TaskValues** column L)  | Truncated normal*        | delta, p, q      | $$mean = delta$$ $$std.dev. = p$$ $$a = mean - p*q$$ $$b = mean + p*q$$ |
+| Seasonality ratio to mean                                     | Uniform                  | mean, p          | $$min = mean - mean*p$$ $$max = mean + mean*p$$ |
+| Minutes per contact <br />(optional; **TaskValues** column O) | Lognormal                | mean, p          | $$sdlog = \sqrt{log[1 + (\frac{p}{mean})^2]}$$ $$meanlog = log(mean)-\frac{log[1 + (\frac{p}{mean})^2)]}{2}$$ |
```
Comment (MeWu-IDM) [inline pacehrh/vignettes/pacehrh.rmd:151]: "Minutes Per Contact" looks slightly different than the original formula (p^2) is now( p/mean)^2?
also I think logarithm is usually followed by parenthesis than the square brackets?

**Review by MeWu-IDM — APPROVED** [review]
formula is more readable! could you double check the one I mentioned, it looks slightly different from the previous version, otherwise all look good to me

File: pacehrh/vignettes/pacehrh.rmd (hunk 1 above)
Comment (rhanIDM) [reply pacehrh/vignettes/pacehrh.rmd:151]: The formula is updated to the more general case. The one we had before is a special case when mean=1.

Comment (rhanIDM) [reply pacehrh/vignettes/pacehrh.rmd:151]: And I agree that log() is more proper notation. Thank you for noticing and making the changes.

**Review by rhanIDM — APPROVED** [review]
Looks good!

**Pull request merged**